packages/mattermost-redux/src/actions/files.ts

```diff
@@ -66,50 +64,6 @@ export function getFilesForPost(postId: string): ActionFunc {
     };
 }
 
-export function uploadFile(channelId: string, rootId: string, clientIds: string[], fileFormData: File, formBoundary: string): ActionFunc {
```

Comment:
this action is replaced with `uploadFile` in this PR https://github.com/mattermost/mattermost-webapp/pull/10169 with uses native xmlhttp